### ISSUE TITLE
Will not find unused images

### ISSUE THREAD

**Issue by julmot** [ryanburgess/grunt-unused#4]
Hi @ryanburgess 

the project structure of my build looks like the following:
![project](https://cloud.githubusercontent.com/assets/8310677/11028546/b62160fe-86bf-11e5-8f14-094f6e204c0e.png)
I am calling grunt-unused with the following arguments:
![arguments](https://cloud.githubusercontent.com/assets/8310677/11028553/c9c83f9c-86bf-11e5-8041-bec8b67d1852.png)
It will say:
![result](https://cloud.githubusercontent.com/assets/8310677/11028563/d93820b4-86bf-11e5-8323-d9c8e716a901.png)
However, there are three files that aren't used in *.html or *.css.

What is going on here?

**Comment by ryanburgess** [comment]
Is there a reason you're passing <%= project.dist.web %>?

This worked for me:

unused: {
      options: {
        reference: 'assets/images/slideshow/',
        directory: ['**/*.css', '**/*.html'],
        days: 30,
        remove: false // set to true to delete unused files from project
      },
    }

**Comment by julmot** [comment]
Yes, this project-object contains my project paths. If I want to change a path I only need to change it at one place.

What could be the reason for my problem?

**Comment by ryanburgess** [comment]
That shouldn't be a problem. You're just passing a variable to the path. I don't see why that would cause problems.

**Comment by julmot** [comment]
I definitely tried every single option but nothing worked for me. I have a handfull images that aren't used in either html nor css, but grunt unused will not find them.
I think in such cases it would be helpful to have a more meaningful console output. Currently the tool just says "0 files unused files:". This doesn't tell you in which path it searched nor which files it checked. Adding this as a "debug" option would be great to allow more informative logs.
